style.css

```diff
@@ -0,0 +1,507 @@
+  font-size: 15px;
+  color: grey;
+}
+
+article {
+  align-content: center;
+}
+
+.carl2 h3 {
+  font-size: 12px;
+}
+
+.carl3 {
+  clear: both;
+}
+
+.carl3 img {
+  width: 80px;
+  height: 80px;
+  float: left;
+  margin-bottom: 15px;
+}
+
+.carl3 p {
+  float: right;
+  font-size: 11px;
+}
+
+footer a:hover {
+  text-decoration: underline;
+}
+
+.carl3 a:hover {
+  text-decoration: underline;
+}
+
+.aside h3 {
+  text-align: left;
+}
+
+body {
+  margin: auto;
+  padding: 0;
+  font-family: nyt-imperial, georgia, 'times new roman', times, serif;
+}
+
+img[src="images/tnyt-logo.png"] {
+  width: 195px;
+  height: 26px;
+  margin-top: 18px;
+}
+
+img[src="images/inflation-945.png"] {
+  max-width: 100%;
+  margin-top: 18px;
+  margin-left: -20px;
+}
+
+section {
+  display: flex;
+  margin-left: 30px;
+}
+
+.body img {
+  width: 847px;
+  height: 476px;
+}
+
+.g-start {
+  font-weight: bold;
+}
+
+figure {
+  position: relative;
+  font-family: 'nyt-franklin', Arial, Helvetica, sans-serif;
+}
+
+.name {
+  text-align: left;
+}
+
+.inflation-img {
+  position: absolute;
+  top: 200px;
+  left: -70px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img1 {
+  position: absolute;
+  top: 200px;
+  left: 120px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img2 {
+  position: absolute;
+  top: 200px;
+  left: 310px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img3 {
+  position: absolute;
+  top: 200px;
+  left: 500px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img4 {
+  position: absolute;
+  top: 530px;
+  left: -70px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img5 {
+  position: absolute;
+  top: 530px;
+  left: 120px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img6 {
+  position: absolute;
+  top: 530px;
+  left: 305px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img7 {
+  position: absolute;
+  top: 530px;
+  left: 500px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation h2 {
+  margin-top: 20px;
+  text-align: left;
+  font-size: 15px;
+}
+
+.inflation-p {
+  font-size: 15px;
+  color: grey;
+  line-height: 18px;
+  margin-top: -10px;
+  text-align: left;
+}
+
+.body p {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+button {
+  height: 30px;
+  align-content: right;
```

Comment:
[HINT]
Right is not a valid value for `align-content`
Check out [here](https://developer.mozilla.org/pt-BR/docs/Web/CSS/align-content)